### PR TITLE
Added Store User Info

### PR DESCRIPTION
**Pull request by TheMonDon**
**Please describe the changes this PR makes and why it should be merged:**
Store Member Info does not work right with users, so I made a user info mod.

**Status**

- [X] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [X] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by almeidx — COMMENTED** [review]
I disagree with this change. I would rather you fix the Store Member Info action to work with users instead of creating a Store User Info separately

**Comment by TheMonDon** [comment]
> I disagree with this change. I would rather you fix the Store Member Info action to work with users instead of creating a Store User Info separately

I think you are wrong as they're two different things they should be kept separate rather than make store member info also have user class

**Comment by TheMonDon** [comment]
> I disagree with this change. I would rather you fix the Store Member Info action to work with users instead of creating a Store User Info separately

I think you are wrong as they're two different things they should be kept separate rather than make store member info also have user class

File: actions/store_user_info_MOD.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+          Store In:<br>
+          <select id="storage" class="round">
+              ${data.variables[1]}
+          </select>
+      </div>
+      <div id="varNameContainer2" style="float: right; width: 60%">
+          Variable Name:<br>
+          <input id="varName2" class="round" type="text"><br>
+      </div>
+  </div>`
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:68]: Use 2 spaces of indentation, and start at 0 (and fix the weird indentation on the select#options)

File: actions/store_user_info_MOD.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    const users = ['Mentioned User', 'Command Author', 'Temp Variable', 'Server Variable', 'Global Variable']
+    const info = ['User Object', 'User ID', 'Username', 'User Status', 'User Avatar URL', 'User Last Message', 'User Last Message ID', 'User Playing Status Name', 'User Custom Status', 'User Discriminator', 'User Tag', 'User Created At', 'User Created Timestamp', 'User Flags List', 'User Client Status']
+    return `${users[parseInt(data.user)]} - ${info[parseInt(data.info)]}`
+  },
+
+  variableStorage (data, varType) {
+    const type = parseInt(data.storage)
+    if (type !== varType) return
+    const info = parseInt(data.info)
+    const dataType = data.info[info]
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:15]: This doesn't make much sense, does it?

File: actions/store_user_info_MOD.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+  subtitle (data) {
+    const users = ['Mentioned User', 'Command Author', 'Temp Variable', 'Server Variable', 'Global Variable']
+    const info = ['User Object', 'User ID', 'Username', 'User Status', 'User Avatar URL', 'User Last Message', 'User Last Message ID', 'User Playing Status Name', 'User Custom Status', 'User Discriminator', 'User Tag', 'User Created At', 'User Created Timestamp', 'User Flags List', 'User Client Status']
+    return `${users[parseInt(data.user)]} - ${info[parseInt(data.info)]}`
+  },
+
+  variableStorage (data, varType) {
+    const type = parseInt(data.storage)
+    if (type !== varType) return
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:13]:
```suggestion
    if (parseInt(data.storage) !== varType) return
```

File: actions/store_user_info_MOD.js [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+    glob.memberChange(document.getElementById('user'), 'varNameContainer')
+  },
+
+  action (cache) {
+    const data = cache.actions[cache.index]
+    const dUser = parseInt(data.user)
+    const varName = this.evalMessage(data.varName, cache)
+    const info = parseInt(data.info)
+    let user = this.getMember(dUser, varName, cache)
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:82]:
```suggestion
    const userType = parseInt(data.user)
    const varName = this.evalMessage(data.varName, cache)
    const info = parseInt(data.info)
    let user = this.getMember(userType, varName, cache)
```

File: actions/store_user_info_MOD.js [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+        }
+        break
+      case 8: // User Custom Status
+        if (this.dest(user.presence, 'activities')) {
+          const status = user.presence.activities.filter((s) => this.dest(s, 'type') === 'CUSTOM_STATUS')
+          result = status && this.dest(status[0], 'state')
+        }
+        break
+      case 9: // User Discriminator
+        result = this.dest(user, 'discriminator')
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:132]:
```suggestion
        result = user.discriminator
```

File: actions/store_user_info_MOD.js [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+        break
+      case 10: // User Tag
+        result = this.dest(user, 'tag')
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:135]:
```suggestion
        result = user.tag
```

File: actions/store_user_info_MOD.js [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+        break
+      case 11: // User Created At
+        result = this.dest(user, 'createdAt')
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:138]:
```suggestion
        result = user.createdAt
```

File: actions/store_user_info_MOD.js [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+        break
+      case 12: // User Created Timestamp
+        result = this.dest(user, 'createdTimestamp')
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:141]:
```suggestion
        result = user.createdTimestamp
```

File: actions/store_user_info_MOD.js [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+        break
+      case 13: // User Flags
+        const flags = this.dest(user, 'flags')
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:144]:
```suggestion
        const flags = user.flags
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/store_user_info_MOD.js [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+      case 1:
+        dataType = 'User ID'
+        break
+      case 2:
+      case 3:
+      case 5:
+      case 7:
+      case 8:
+      case 14:
+        dataType = 'Text'
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:28]: It's a list. Just merge it with case 13

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**